src/EventService.Validation/Event/CreateEventRequestValidator.cs

```
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Linq;
+using FluentValidation;
+using LT.DigitalOffice.EventService.Broker.Requests.Interfaces;
+using LT.DigitalOffice.EventService.Data;
+using LT.DigitalOffice.EventService.Data.Interfaces;
+using LT.DigitalOffice.EventService.Models.Dto.Requests.Event;
+using LT.DigitalOffice.EventService.Validation.Event.Interfaces;
+using LT.DigitalOffice.Kernel.Extensions;
+using Microsoft.AspNetCore.Http;
+
+namespace LT.DigitalOffice.EventService.Validation.Event;
+
+public class CreateEventRequestValidator : AbstractValidator<CreateEventRequest>, ICreateEventRequestValidator
+{
+  public CreateEventRequestValidator(
+    IUserService userService,
+    IHttpContextAccessor contextAccessor,
+    ICategoryRepository categoryRepository)
+  {
+    RuleFor(ev => ev.Name)
+      .MaximumLength(150)
+      .WithMessage("Name should not exceed maximum length of 150 symbols");
+
+    When(ev => !string.IsNullOrEmpty(ev.Description), 
```

use IsNullOrWhiteSpaces

IsNullOrEmpty catches null + ""
IsNullOrWhiteSpaces catches null + "" + "     "

---

Done